spec/services/authors_coauthor_countries_report_generator_spec.rb

```diff
@@ -3,12 +3,14 @@
 require 'rails_helper'
 
 RSpec.describe AuthorsCoauthorCountriesReportGenerator do
-  subject(:report) { described_class.generate(org_uri: organization.uri, start_year: start_year, end_year: end_year) }
+  subject(:report) { described_class.generate(org_uri: organization_uri, start_year: start_year, end_year: end_year) }
 
   let(:start_year) { '2016' }
 
   let(:end_year) { '2018' }
 
+  let(:organization_uri) { organization ? organization.uri : nil }
```

Comment:
in ruby you can do `organization&.uri` http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/